jsapp/js/utils.ts

```diff
@@ -577,3 +578,22 @@ export function removeDefaultUuidPrefix(uuid: string) {
 export function matchUuid(uuidA: string, uuidB: string) {
   return addDefaultUuidPrefix(uuidA) === addDefaultUuidPrefix(uuidB)
 }
+
+export function createDateQuery(startDate: string, endDate: string) {
```

Comment:
Should this be returning `MongoQuery`?
```suggestion
export function createDateQuery(startDate: string, endDate: string): MongoQuery {
```